operator/pkg/resources/lb_service.go

```diff
@@ -143,3 +143,73 @@ func (r *LoadBalancerServiceResource) getAnnotation() map[string]string {
 	}
 	return ext.External.Bootstrap.Annotations
 }
+
+// RenderLoadBalancerService renders a LoadBalancer service
+func RenderLoadBalancerService(
```

Comment:
It seems this is more than just a "loadbalancerservice" perhaps this function should be called `RenderBootstrapService`?